tmail-backend/apps/distributed/src/test/java/com/linagora/tmail/james/app/DistributedServerWithOpenPaasRabbitMqConfiguredTest.java

```diff
@@ -0,0 +1,46 @@
+package com.linagora.tmail.james.app;
+
+import org.apache.james.JamesServerBuilder;
+import org.apache.james.JamesServerExtension;
+import org.apache.james.SearchConfiguration;
+import org.apache.james.backends.redis.RedisExtension;
+import org.apache.james.utils.GuiceProbe;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.google.inject.multibindings.Multibinder;
+
+import com.linagora.tmail.OpenPaasModuleChooserConfiguration;
+import com.linagora.tmail.combined.identity.LdapExtension;
+import com.linagora.tmail.combined.identity.UsersRepositoryClassProbe;
+import com.linagora.tmail.combined.identity.UsersRepositoryModuleChooser;
+import com.linagora.tmail.module.LinagoraTestJMAPServerModule;
+
+public class DistributedServerWithOpenPaasRabbitMqConfiguredTest {
+    @RegisterExtension
+    static JamesServerExtension
+        testExtension = new JamesServerBuilder<DistributedJamesConfiguration>(tmpDir ->
+        DistributedJamesConfiguration.builder()
+            .workingDirectory(tmpDir)
+            .configurationFromClasspath()
+            .searchConfiguration(SearchConfiguration.openSearch())
+            .usersRepository(UsersRepositoryModuleChooser.Implementation.COMBINED)
+            .eventBusKeysChoice(EventBusKeysChoice.REDIS)
+            .openPassModuleChooserConfiguration(OpenPaasModuleChooserConfiguration.ENABLED)
+            .build())
+        .server(configuration -> DistributedServer.createServer(configuration)
+            .overrideWith(new LinagoraTestJMAPServerModule())
+            .overrideWith(binder -> Multibinder.newSetBinder(binder, GuiceProbe.class).addBinding().to(UsersRepositoryClassProbe.class)))
```

Comment:
Is this module override needed?

Reply:
Probably not, we don't use jmap in this test. The test passes even after removing the module. It is hard to tell for sure which modules or extensions are needed and which are not.